core/src/main/java/ar/com/wolox/wolmo/core/util/Logger.kt

```diff
@@ -0,0 +1,179 @@
+package ar.com.wolox.wolmo.core.util
+
+import android.util.Log
+import javax.inject.Inject
+
+class Logger @Inject constructor() {
+
+    private var tag: String? = null
+
+    /**
+     * Sets the tag for this logger.
+     *
+     * @param tag Used to identify the source of a log message.  It usually identifies
+     * the class or activity where the log call occurs.
+     */
+    fun setTag(tag: String) {
+        this.tag = tag
+    }
+
+    /**
+     * Send a [Log.VERBOSE] log message.
+     * The tag used is the one defined by [.setTag].
+     * @param msg The message you would like logged.
+     */
+    fun v(msg: String): Int = v(tag, msg)
```

Comment:
We could avoid all this `: Int` and let Kotlin infer it.

Reply:
ok :+1: